lib/app.rb

```diff
@@ -139,15 +139,31 @@ def get_key_with_tls_version(key, version)
 end
 
 def redis_client_tls(version='TLSv1')
-  if version == 'TLSv1_3'
-    return @client_tls_13 ||= RedisTLS13.new(
-      host: redis_credentials.fetch('host'),
-      port: redis_credentials.fetch('tls_port'),
-      password: redis_credentials.fetch('password')
-    )
+  if redis_credentials.key?('sentinels')
+    if version == 'TLSv1_3'
+      client_redis_details_from_sentinel = SentinelTLS13.new(
```

Comment:
(cosmetic) this code snippet can be simplified as below

```ruby
master_redis = SentinelTLS13.new(
  host: redis_credentials.fetch('sentinels').first['host'],
  port: redis_credentials.fetch('sentinels').first['tls_port']
).get_redis_instance
```

Reply:
DOne